torch/_dynamo/eval_frame.py

```diff
@@ -884,6 +897,21 @@ def graph_with_interpreter(*args):
         graph,
     ).transform()
 
+    # Store constraints and inputs as metadata for user passes, e.g. turn constraints to runtime check
+    new_graph.meta["example_inputs"] = example_inputs
+    new_graph.meta["input_shape_constraints"] = (
+        [constraint.serializable_spec for constraint in constraints]
+        if constraints
+        else None
+    )
+    new_graph.meta["inline_constraints"] = {
```

Comment:
@tugsbayasgalan BTW, the metadata stored in exported graph module won’t be preserved after `torch.store()/load()`. My understanding is that it’s expected as those are not parameters and buffer. If the exported graph gets saved/serialized prior to run the pass turning it into runtime check, they lost it.

Reply:
Yep that is fine. Primary reason for store and load to work is to not break other people's flow. I will directly access them, so should be good